src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java

```diff
@@ -64,4 +64,16 @@ private static Document mapScannedDocument(ScannedDocument doc) {
             );
         }
     }
+
+    private static String getFileName(ScannedDocument doc) {
+        if (doc.fileName == null) {
+            if (doc.url == null) {
+                return null;
+            } else {
+                return doc.url.documentFilename;
+            }
+        } else {
+            return doc.fileName;
```

Comment:
are we sure doc.fileName == doc.url.documentFilename. ??

Reply:
No. But this does not matter for us - we are checking if not both are null